improver_tests/acceptance/test_combine.py

```diff
@@ -110,3 +110,24 @@ def test_midpoint(tmp_path):
     ]
     run_cli(args)
     acc.compare(output_path, kgo_path)
+
+
+def test_combine_broadcast(tmp_path):
+    """Test combining precipitation realizations with phaseprob"""
+    kgo_dir = acc.kgo_root() / "combine/broadcast"
+    kgo_path = kgo_dir / "kgo.nc"
+    inputs = [kgo_dir / f for f in ["input_larger_cube.nc", "input_smaller_cube.nc"]]
+    output_path = tmp_path / "output.nc"
+    args = [
+        *inputs,
+        "--operation",
+        "multiply",
+        "--broadcast-to-coord",
+        "threshold",
```

Comment:
Is this ever likely to be anything other than "threshold"?

Reply:
I wouldn't have said it was ever going to be needed just a month ago. I don't know of any other use-cases yet, but I imagine that future multi-variate probabilities will need this. Even with that thought, I can't see a reason why it would be anything other than "threshold".

Reply:
In which case, the advice I generally get on complexity is that we're better off with boolean than string arguments - easier to use, harder to get wrong.  Do you think the CLI and plugin argument could be "broadcast_across_thresholds" (or equivalent), which has a simple `True / False` value?  This simplifies and tidies the interface.

Reply:
Ok. I've done that.